src/main/java/org/spring/FullVersion/FamilyService.java

```diff
@@ -124,5 +125,19 @@ public void addPet(int index, Pet pet) {
         }
     }
 
+    public void saveFamilies() {
+        familyDao.saveData(familyDao.getAllFamilies());
+        Logger.info("Families successfully saved to file.");
 
-}
+
+    }
+
+    public void loadFamilies() {
+        List<Family> loadedFamilies = familyDao.loadData();
+        if (loadedFamilies != null) {
+            familyDao.saveData(loadedFamilies);
+            Logger.info("Families successfully loaded from file.");
+        }
+
+    }
+}
```

Comment:
you can catch possible exceptions here

Reply:
I don't know how to do it
@sabuhihuseynov